test/unit/alignment/pairwise/align_result_selector_test.cpp

```diff
@@ -29,14 +29,6 @@ TEST(alignment_selector, align_result_selector_with_list)
     using seq1_t = std::vector<dna4>;
     using seq2_t = std::list<dna4>;
 
-    { // test case I
-        auto cfg = align_cfg::edit;
-        using _t = alignment_result<typename detail::align_result_selector<seq1_t, seq2_t, decltype(cfg)>::type>;
-
-        EXPECT_TRUE((std::is_same_v<decltype(std::declval<_t>().id()), uint32_t>));
-        EXPECT_TRUE((std::is_same_v<decltype(std::declval<_t>().score()), int32_t>));
-    }
-
     { // test case II
         auto cfg = align_cfg::edit | align_cfg::result{with_score};
```

Comment:
can't you make this one case I and ... 

(or you have to change the numbering `// test case II` -> `// test case I`, `// test case III`)